server/pkg/schema/field.go

```diff
@@ -20,6 +20,7 @@ type Field struct {
 	unique       bool
 	multiple     bool
 	required     bool
+	isTitle      *bool
```

Comment:
```suggestion
	isTitle      bool
```

The field should have no state other than being titled or not titled.

Reply:
Done!